### PR TITLE
base: split out a ubuntu 16 version

### PR DESCRIPTION
**Pull request by lamping7**
We need this to update the ros-kinetic container. Solves entrypoint, user and xvfb issues.

I added it to docker hub: https://hub.docker.com/r/px4io/px4-dev-base-xenial

### CODE REVIEW

**Comment by dagar** [comment]
How should we have done this from the beginning?

What about a containers-ubuntu repo with branches per version (no master)?

**Comment by lamping7** [comment]
The downside to that is that we have to maintain the `entrypoint` across branches. I thought about updating the existing `ubuntu16.04` branch, but didn't like that downside.

File: docker/px4-dev/Dockerfile_ros-kinetic [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # PX4 ROS development environment
 #
 
-FROM px4io/px4-dev-base:ubuntu16.04
+FROM px4io/px4-dev-base-xenial:pr-base-ubuntu16
```
Comment (lamping7) [inline docker/px4-dev/Dockerfile_ros-kinetic]: This will need to change once tagged

**Comment by mrivi** [comment]
@dagar @lamping7 could we find a solution for this? Maybe talking about it in the dev call?

**Comment by lamping7** [comment]
I can rebase if we want to move forward with this.

**Comment by lamping7** [comment]
rebased

**Comment by dagar** [comment]
I'll merge this as soon as we can get it through CI cleanly.

I'm thinking we should abandon the simple naming and include versions everywhere.

**Comment by dagar** [comment]
Let's also discard the ubuntu16.04 branch once this is in.

**Pull request merged**